test/external/testcorenrn/CMakeLists.txt

```diff
@@ -72,6 +72,15 @@ foreach(test ${spike_comparison_tests})
       OMP_NUM_THREADS=1 HOC_LIBRARY_PATH=. ${MPIEXEC_NAME} ${MPIEXEC_NUMPROC_FLAG} ${num_ranks}
       ${MPIEXEC_OVERSUBSCRIBE} ${MPIEXEC_PREFLAGS} special ${MPIEXEC_POSTFLAGS} -mpi -c
       arg_tstop=100 test${test}.hoc)
+
+  # bbcore test generated gids starting at zero, we need to enable this feature in libsonata
+  # otherwise an assert fail inside libsonata-report
+  if(NOT "${test}" STREQUAL "vecevent")
```

Comment:
Why does `vecevent` have to be treated differently?

Reply:
Because the gid created are above 0

Reply:
It works without, so I prefer to keep it without.

Reply:
We need to be more verbose for this `if`. Is there any ticket or code we can point at ?